CHANGELOG.md

```diff
@@ -496,4 +496,4 @@
 
 
 
-\* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
```

Comment:
@karawoo could you please help figure out why this changed? it's the only reason we haven't merged this - wanted to avoid making changes to the changelog outside of the release process.

Reply:
I have no idea, but the content is not actually changed at all, so I'm not too worried about it. I can rebase this branch to just keep the change to `experimentalData.json` if we want to prevent it from looking as though the changelog has been changed.